ethcontract-mock/src/lib.rs

```diff
@@ -482,6 +482,12 @@ pub struct Expectation<P: Tokenize + Send + 'static, R: Tokenize + Send + 'stati
     _ph: PhantomData<(P, R)>,
 }
 
+// Allow not requiring `must_use` as `Expectation`s have side effects, but
+// return `Self` as a convenience for chaining operations. Note that this lint
+// is on nighlty only, so we also need to allow `unknown_lints` here to silence
+// warnings on all Rust versions we build for.
+#[allow(unknown_lints)]
```

Comment:
I personally think this is fine (and not failing nightly could be worth it to catch other upcoming issues). Is the tradeoff that we might accidentally allow other unknown lints here (which I think is not that severe)?

Reply:
Yes that is the trade-off.